lib/tasks/batch_connect.rake

```diff
@@ -0,0 +1,31 @@
+namespace :batch_connect do
+  desc "Generate new batch connect session"
+  task new_session: :environment do
+    # Read in user settings
+    app = ENV["BC_APP_TOKEN"] || abort("Missing environment variable BC_APP_TOKEN")
+    fmt = ENV["BC_RENDER_FORMAT"]
+    ctx = if !$stdin.tty?
```

Comment:
Actually I would need to test whether the solution I provided would still skip the stdin reading if it was launched from a process without a tty.